src/nspawn/nspawn-mount.c

```diff
@@ -766,6 +771,12 @@ static int mount_bind(const char *dest, CustomMount *m) {
                         return log_error_errno(r, "Read-only bind mount failed: %m");
         }
 
+        if (idmapped) {
```

Comment:
this should be conditionalized further, i.e.that userns is actually on, and a uid map defined